aya/src/maps/mod.rs

```diff
@@ -174,6 +177,16 @@ pub enum MapError {
         io_error: io::Error,
     },
 
```

Comment:
I think you can re-use SyscallError here. No need for an extra error type.

Reply:
Done, used syscallError and removed the `code` field